src/lib/esnext.collection.d.ts

```diff
@@ -1,15 +1,3 @@
-interface MapConstructor {

```

Comment:
We should probably include a lib reference to es2024.collection.d.ts so as to not break anyone currently using `"lib": [..., "esnext.collection.d.ts"]` for this.